_episodes/03-transform.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Plotting and Tabular Data"
-teaching: 65
-exercises: 20
+teaching: 50
+exercises: 5
```

Comment:
Substantial decrease here in the exercise timing. Correct?

Reply:
Yes, because this episode was split into two episodes and some of the exercises moved to the new episode.

Reply:
Of course!